### DISCUSSION TITLE
Support for Epson L3050

### DISCUSSION THREAD

**Discussion by Hypper-done** [Ircama/epson_print_conf#30] [Ideas]
Hello, would it be possible to add support for Epson L3050 ? Thank you in advance !

**Comment by Ircama** [comment]
This repository is focused on Python code development, I'm sorry but I have no time to provide usage support and to add/test new printers.

You might try [adding it yourself](https://github.com/Ircama/epson_print_conf/?tab=readme-ov-file#example-to-integrate-new-printers) (at your risk).

**Comment by GHYaBoi** [comment]
Check out atufi's reinkpy project on Codeberg. Worked perfectly for me

**Comment by Ircama** [comment]
https://github.com/Ircama/epson_print_conf#how-to-import-an-external-printer-configuration-db
